### DISCUSSION TITLE
What about a loop command?

### DISCUSSION THREAD

**Discussion by Lliakos** [jagrosh/MusicBot#647] [Ideas]
What about a loop command? So when you play a music thats oonly one minute you woulnt have to spam it and just loop it

**Comment by jagrosh** [comment]
Are you talking about [the repeat command](https://github.com/jagrosh/MusicBot/wiki/Full-Command-Reference#repeat-on--off)?

**Reply by Ed1013** [reply]
Repeat command loops the whole queue, a command to repeat the playing song would be very nice
